src/field_theory/splitting_field.lean

```diff
@@ -402,11 +402,33 @@ begin
   exact eq_of_monic_of_associated hprodmonic hmonic hassoc
 end
 
+lemma prod_multiset_X_sub_C_of_monic_of_roots_card_eq {K : Type*} [comm_ring K] [is_domain K]
```

Comment:
I'd make the field version `private` and move it on top of this result.

Reply:
What do you mean by on top? They're already right next to each other?

Reply:
oops, I didn't expand the arrows on github and thought there was stuff between, but it's just a long proof

Reply:
Ok cool, I'll make them private though